image/image.go

```diff
@@ -87,6 +88,18 @@ func validate(w walker, refs []string, out *log.Logger) error {
 		if err := m.validate(w); err != nil {
 			return err
 		}
+
+		ml, err := findManifestList(w, d)
+		if err == nil {
+			if err := ml.validate(w); err != nil {
+				return err
+			}
+		} else {
+			if !strings.Contains(string(err), "manifestlist not found") {
```

Comment:
Yes, I was in accordance with the idea to write, I first made a judgment on the error message, if it is because there is no manifest lis and return, not error out.（There is one! The symbol follows the if）